dispatch/static/manager/src/js/index.js

```diff
@@ -26,7 +28,14 @@ const store = createStore(
     loadingBar: loadingBarReducer,
     modal: modalReducer
   }),
-  applyMiddleware(thunk, router, promiseMiddleware(), loadingBarMiddleware())
+  applyMiddleware(
+    thunk,
+    router,
+    promiseMiddleware(),
+    loadingBarMiddleware({
+      promiseTypeSuffixes: ['PENDING', 'FULFILLED', 'REJECTED'],
```

Comment:
Okay, I'll make an issue before the next meeting highlighting which ones need to be refactored. I'm going to exclude any reducer that needs to be changed as part of implementing a page/component that we already have an issue for (ie. not going to include the people reducer, for example since its already going to be changed when we implement the people page)